src/api/.rubocop.yml

```diff
@@ -118,6 +118,7 @@ Style/MethodCallWithArgsParentheses:
     - assert_select
     - assert_xml_tag
     - authorize
+    - be
```

Comment:
Not sure if this belongs to the same commit

Reply:
I moved it to another commit